package.json

```diff
@@ -67,7 +67,7 @@
     "typescript": "4.7.4"
   },
   "resolutions": {
-    "undici": "5.8.2",
+    "undici": "5.9.1",
```

Comment:
Things appear to be working, but this change is in the `resolutions`. Do we want to update it or does this need to stay the same 🤔

Reply:
we can leave these unless there is a new security issue